src/cc_catalog_airflow/dags/util/loader/sql.py

```diff
@@ -246,3 +265,25 @@ def _get_load_table_name(
         load_table_name_stub=LOAD_TABLE_NAME_STUB,
 ):
     return f'{load_table_name_stub}{identifier}'
+
+
+def _get_malformed_row_in_file(error_msg):
+    error_list = error_msg.splitlines()
+    copy_error = error_list[3]
+    assert copy_error.startswith('COPY'), copy_error
```

Comment:
We can *reduce* (but not eliminate) dependency on the error message format with:
```suggestion
    copy_error = next(
        (line for line in error_list if line.startswith('COPY')), None
    )
    assert copy_error is not None
```
This removes the need to care about which line the message is on

Reply:
Note if there's more than one line starting with `COPY`, then this will take the first.